pkg/api/validation/validation.go

```diff
@@ -3198,10 +3198,21 @@ func ValidateConfigMap(cfg *api.ConfigMap) field.ErrorList {
 		for _, msg := range validation.IsConfigMapKey(key) {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("data").Key(key), key, msg))
 		}
+		if _, isValue := cfg.BinaryData[key]; isValue {
+			allErrs = append(allErrs, field.Duplicate(field.NewPath("data").Key(key), key))
```

Comment:
So, should I remove this in this case? Validation is also done in kubectl https://github.com/kubernetes/kubernetes/pull/33549/files#diff-c52de27ddc6a3a367a09a08ec5122295R222 and in this case should be fine to have it only this place